crates/hir-def/src/lib.rs

```diff
@@ -96,6 +95,8 @@ use crate::{
     signatures::VariantFields,
 };
 
+pub use crate::db::DefDatabase;
```

Comment:
The `db` module is public, no need for this.

Reply:
whoops, my bad.